coin_change_2.py

```diff
@@ -0,0 +1,34 @@
+'''
+Implementation: 
+Initial array:
+1 0 0 0 0 0
+On Coin 1, start with index 1:
+1 1 1 1 1 1
+On Coin 2, start with index 2 and add existing value with a - 2 (2 steps back) and keep other same:
+1 1 2 2 3 3
+On Coin , start with index 5 and add exisiting value with a - 5 (5 steps back) and keep others same:
+1 1 2 2 3 4
+
+Final answer = arr[amount]
+
+Time complexity: O(n)
+Space complexity: O(n)
```

Comment:
I believe the TC of this solution would be O(amount * coins).

Be more specific with your SC. What's _n_ in your SC?